pkg/apis/hive/v1alpha1/syncset_types.go

```diff
@@ -116,12 +118,45 @@ type SyncCondition struct {
 	Message string `json:"message,omitempty"`
 }
 
+// SelectorSyncSetObjectStatus describes the status of resources created or patches that
+// have been applied from a SyncSet.
+type SelectorSyncSetObjectStatus struct {
```

Comment:
@abutcher so sorry for the very late comment, but just noticed this since you removed the namespace field. The SelectorSyncSetObjectStatus and SyncSetObjectStatus structs are identical. We could get rid of one. (Just SyncSetObjectStatus).

The other thing I noticed is that we should likely just name the fields like `Resources` and `Patches` instead of `SyncResources` and `SyncPatches`. The sync part is already implied in the type.